### PR TITLE
Fix 3760 more visible subject email on review assignment

### PR DESCRIPTION
**Pull request by evyncke**
Fix issue 3760 by a new subject line, e.g.,
`Subject: For Benno Overeinder, Intdir Last Call review assigned: draft-ietf-add-dnr`

Also, send the email to all secretaries even to the requestor (so they can 'see' what is automatically generated).

### CODE REVIEW

File: client/agenda/AgendaMobileBar.vue [hunk 1]
```diff
@@ -31,7 +31,7 @@ import {
 } from 'naive-ui'
 
 import { useAgendaStore } from './store'
-import { useSiteStore } from '../shared/store';
+import { useSiteStore } from '../shared/store'
```
Comment (larseggert) [inline client/agenda/AgendaMobileBar.vue:34]: Superfluous change?

Comment (rjsparks) [reply client/agenda/AgendaMobileBar.vue:34]: There are many files in here that shouldn't be - I suspect Eric was based in the wrong place. I'll work with him to pull out the parts of this that were meant to change.

Comment (evyncke) [reply client/agenda/AgendaMobileBar.vue:34]: Indeed, I tried to start from feat/tzaware rather than main but I failed.

Comment (evyncke) [reply client/agenda/AgendaMobileBar.vue:34]: Will redo the changes in a fresh PR

**Comment by evyncke** [comment]
This is the plan for this week, so good move

**Pull request closed without merging**